tests/fixtures/event/base.py

```diff
@@ -0,0 +1,392 @@
+"""
+Base class definitions for the event fixture
+"""
+from faker import Faker
+
+# Faker.seed(0)
+FAKE = Faker()
+
+
+class BaseEventField:
+    """BaseEventField Interface.
+    The Event Field is just a wrapper around it's value which is obtained
+    by calling it's generator function.
+    """
+
+    # pylint: disable=too-many-instance-attributes
+    # pylint: disable=too-many-arguments
+    def __init__(
```

Comment:
pylint complains about too many attributes / arguments.
Do you prefer if i wrap some arguments in a Data class instead?
On one side this will make this part cleaner but on the other side for each EventField creation more code will have to be written. Example:
current:
`self.session = FreeEventField(FAKE.md5, emptiable_str=True, raw_output=False)`
with dataclass:
`self.session = FreeEventField(FAKE.md5, properties=EventFieldProperties(emptiable_str=True), raw_output=False)`

Reply:
Using a dataclass is cleaner IMO.

Reply:
Thanks, will be done